backend/back/package.json

```diff
@@ -42,6 +42,7 @@
     "axios": "^1.5.1",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
+    "crypto": "^1.0.1",
```

Comment:
I don't understand why you are putting this in the package.json, this is now a builtin: https://www.npmjs.com/package/crypto

Reply:
Fixed